src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -100,14 +102,19 @@ export const useSendThorTx = ({
     return ['withdrawLiquidity', 'withdrawSavers'].includes(action)
   }, [action])
 
+  // Either a fall through of the passed dustAmountCryptoBaseUnit, or the default dust amount for that feeAsset
   // @TODO: test this with RUNEPool, might not work properly due to mapping for LPs
-  const dustAmountCryptoBaseUnit = useMemo(() => {
-    return THORCHAIN_SAVERS_DUST_THRESHOLDS_CRYPTO_BASE_UNIT[feeAsset?.assetId ?? ''] ?? '0'
-  }, [feeAsset])
+  const dustAmountCryptoBaseUnitOrDefault = useMemo(() => {
```

Comment:
suggestion - not sure how useful the `OrDefault` naming is and can reduce the diff a bit

Reply:
Was shooting for overly descriptive here but happy to remove it and reduce diff-size!

1fbaa783a4867463b7dc3ed263c24331d5f332b4